torch/testing/_internal/common_methods_invocations.py

```diff
@@ -17392,14 +17392,14 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
         "_refs.sub",
         torch_opinfo_name="sub",
         # https://github.com/pytorch/pytorch/issues/76944
-        supports_two_python_scalars=False,
+        supports_two_python_scalars=True,
         supports_one_python_scalar=True,
     ),
     ElementwiseBinaryPythonRefInfo(
         "_refs.true_divide",
         torch_opinfo_name="true_divide",
         # https://github.com/pytorch/pytorch/issues/76944
-        supports_two_python_scalars=False,
+        supports_two_python_scalars=True,
```

Comment:
If these were not explicitly specified would the values be inherited?

Reply:
default value for supports_two_python_scalars is False